docs/user-guide/manage-environments.md

```diff
@@ -178,7 +178,25 @@ To inspect the deployed application in KubeRocketCI portal, take the following s
 
     ![Inspect application](../assets/user-guide/environments/application_terminal.png "Inspect application")
 
+### Delete Application
+
+There are two buttons in the environment which can be used in the portal:
+
+  ![Clean button](../assets/operator-guide/edp-portal-clean-application.png "Clean button")
+
+  * **Delete** - Deletes selected applications and associated resources.
+  * **Clean** - This button is useful when you need to perform custom cleanup actions, e.g. delete cloud resources, database, rollback transactions, etc.
+
+The **Delete** button works best when your application doesn't require deploying spicific resources, such as databases and external applications, or configuring the environment in a specific way.
+
+The **Clean** button triggers a delete pipeline that incorporates any custom logic you define. This option is best suited for applications that require complex configurations for proper operation, ensuring that related resources with the same lifecycle as the application are deleted when the application is no longer needed.
+
+KubeRocketCI offers a simple clean pipeline out of the box. So initially, there is no difference between the buttons, because in this case, they do the same actions. Things change when you create your own delete pipeline to manually configure the **Clean** button logic.
```

Comment:
KubeRocketCI provides an intuitive and streamlined pipeline by default. Initially, there is no distinction between the "Delete" and "Clean" buttons, as both perform identical actions. This default behavior encompasses the straightforward deletion of applications, which involves the uninstallation of the associated Helm chart. However, this functionality evolves once a custom delete pipeline is established, enabling manual configuration of the logic behind the "Clean" button. This customization allows for a more tailored approach to managing application lifecycles within KubeRocketCI.